lib/notesclub_web/router.ex

```diff
@@ -44,7 +44,7 @@ defmodule NotesclubWeb.Router do
     pipe_through :browser
 
     # Used for uptime monitoring and zero-downtime deploys
```

Comment:
We can get rid of the comment. We now monitor `/` instead of `/ok` and deploys no longer need it.
```suggestion
```

Reply:
Done!